docs/README.md

```diff
@@ -1,21 +1,33 @@
-# Documentation Area
-Everything related to the IOTA Wallet library.
+# Website
 
-## Run locally
-You can run a local server to run the documentation page.
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-### Dependencies
+## Installation
 
-[mdBook]https://github.com/rust-lang/mdBook is a utility to create modern online books from Markdown files.
+```console
+yarn install
+```
+
+## Local Development
 
-```bash
-cargo install mdbook
+```console
+yarn start
 ```
 
-### Run it
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Build
+
+```console
+yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+## Deployment
 
-```bash
-mdbook serve
+```console
+GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
```

Comment:
We're using a Github Action for deploying this page on Github Pages. Here is an example for docusaurus: https://github.com/iota-community/Community-Governance/blob/main/.github/workflows/gh-pages.yml

Means - we dont need this here  - just add "Now you can visit `http://localhost:3000`" again to watch it on the local machine :)